### PR TITLE
Briefly enumerate changes in Docker image announcement

### PR DESCRIPTION
**Pull request by jakirkham**
Try to briefly enumerate the changes at a high-level to give readers a better sense of what they are.

<hr>

PR Checklist:

- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [ ] if you are adding a new page under `docs/` or `community/`, you have added it to the sidebar in the corresponding `_sidebar.json` file
- [ ] put any other relevant information below

### CODE REVIEW

**Comment by jakirkham** [comment]
@h-vetinari when you have a moment, could you please take a look at this?

Am not attached to this exact wording or structure, but am interested in capturing these high-level points up top. So if you have thoughts on improving it, please let me know 🙂

File: news/2024-11-22-new-images.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # Updating our default docker images
 
-TL;DR: We've updated our default docker images to be based on alma9, and we've
-consolidated our image names to follow a consistent pattern:
+TL;DR: We have made some updates to our Docker images and build time GLIBC selection. 
+
+1. We've updated our default docker images to be based on alma9
+2. Our default GLIBC version during builds remains 2.17 (ensuring broad compatibility).
+3. When necessary, it is now possible to build against a newer GLIBC.
```
Comment (h-vetinari) [inline news/2024-11-22-new-images.md:7]: These two haven't really changed? The only change there was decoupling stdlib from the docker_image (for non-cuda recipes, this worked hassle-free already before).

Here's a suggestion
```suggestion
2. It is now easier to override `c_stdlib_version` (especially for CUDA-enabled feedstock), though our baseline of 2.17 hasn't changed
3. Where necessary, you can more easily switch images by setting `os_version: ...` in `conda-forge.yml`
```

**Review by h-vetinari — COMMENTED** [review]
Thanks for the suggestion John! Happy to improve the text with a more detailed list of changes in the TL;DR. I've made a suggestion for improvement, but otherwise this LGTM!

**Comment by h-vetinari** [comment]
pre-commit.ci autofix

**Comment by jakirkham** [comment]
Thanks Axel! 🙏

Sounds reasonable. Have included it. Please let me know if there's anything else

**Review by h-vetinari — APPROVED** [review]

**Pull request merged**